### ISSUE TITLE
[a11y] Cannot open Exchange Out

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1379]
**Describe the bug**
This is occuring on the `freshfish` db

**To Reproduce**
1. Go to Interactions
2. Click on Exchange Out
3. See error

**Screenshots**
![image](https://user-images.githubusercontent.com/37256050/166499928-f40e8884-3719-4e8f-9982-28b86ea733bd.png)

**Desktop (please complete the following information):**
 - macOS, Edge
